Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerMentionCollectionViewCell.swift

```diff
@@ -5,61 +5,74 @@
 import StreamChat
 import UIKit
 
+/// A View that is embed inside `UICollectionViewCell`  which shows information about user which we want to tag in suggestions
 public typealias ChatMessageComposerMentionCellView = _ChatMessageComposerMentionCellView<NoExtraData>
 
+/// A View that is embed inside `UICollectionViewCell`  which shows information about user which we want to tag in suggestions
 open class _ChatMessageComposerMentionCellView<ExtraData: ExtraDataTypes>: View, UIConfigProvider {
     // MARK: Properties
```

Comment:
nit: Let's remove noise like this, especially for such small classes

Reply:
Deleted.

Reply:
It's not

Reply:
Now it is.